README.md

```diff
@@ -50,9 +69,13 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
   @import "bourbon/bourbon";
   ```
 
-  It’s not recommended to add or modify the Bourbon files so that you can update them easily.
+  It’s not recommended that you modify the Bourbon files so that you can update
```

Comment:
Great point. I’d like to take a stab at even further refinement:

> It’s not recommended that you modify Bourbon’s files directly as it will make updating to future versions difficult, by overwriting your custom changes or causing merge conflicts.